gax-grpc/src/test/java/com/google/api/gax/grpc/ChannelPoolTest.java

```diff
@@ -158,4 +164,69 @@ public void run() {
       Truth.assertThat(count.get()).isAnyOf(expectedCount, expectedCount + 1);
     }
   }
+
+  // Test channelPrimer is called same number of times as poolSize if executorService is set to null
+  @Test
+  public void channelPrimerShouldBeCalledOnce() throws IOException {
+    ChannelPrimer mockChannelPrimer = Mockito.mock(ChannelPrimer.class);
+    ManagedChannel channel1 = Mockito.mock(ManagedChannel.class);
+    ManagedChannel channel2 = Mockito.mock(ManagedChannel.class);
+
+    ChannelPool.create(
+        2, new FakeChannelFactory(Arrays.asList(channel1, channel2), mockChannelPrimer));
+    Mockito.verify(mockChannelPrimer, Mockito.times(2))
+        .primeChannel(Mockito.any(ManagedChannel.class));
+  }
+
+  // Test channelPrimer is called periodically, if there's an executorService
+  @Test
+  public void channelPrimerIsCalledPeriodically() throws IOException, InterruptedException {
+    ChannelPrimer mockChannelPrimer = Mockito.mock(ChannelPrimer.class);
+    ManagedChannel channel1 = Mockito.mock(RefreshingManagedChannel.class);
+    ManagedChannel channel2 = Mockito.mock(RefreshingManagedChannel.class);
+    ManagedChannel channel3 = Mockito.mock(RefreshingManagedChannel.class);
+
+    final List<Runnable> channelRefreshers = new ArrayList<>();
+
+    ScheduledExecutorService scheduledExecutorService =
+        Mockito.mock(ScheduledExecutorService.class);
+    Mockito.when(
```

Comment:
(applies to other similar calls in tests in this PR, like `testWithPrimeChannel()`) Can you please format it in an easier to read way (i.e. using intermediate variables, so this stops being one huge call to `Mockito.when(`?

I understand that it is subjective, but it is kind of hard to read when lots of logic is inside multiple nested `()`  of method calls (i.e. you see many lines of logic in place, where a simple method parameter is expected)).